tasks/util/postcss.ts

```diff
@@ -5,6 +5,7 @@ const postCssImport = require('postcss-import');
 const postCssNext = require('postcss-cssnext');
 const postCssModules = require('postcss-modules');
 const umdWrapper = require('./umdWrapper');
+const pkgUp = require('pkg-up');
```

Comment:
Take a look at https://github.com/dojo/grunt-dojo2/blob/cd39e810f8005d76bcd6c3578eb570ac503551f6/index.ts#L97

Other tasks have `packagejson` passed into them as the second argument. I believe you should change the postcss task to do the same and then pass it into this function

Reply:
Ok didn't realise, have updated it to use `packagejson` now.